Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java

```diff
@@ -134,46 +199,79 @@ public void initDefaultCommand() {
 		setDefaultCommand(new TeleopDrive());
 	}
 
+	/**
+	 * Disables the turnPID PIDController used for turning
+	 */
 	public void disableTurnPid() {
 		turnController.disable();
 	}
 
+	/**
+	 * Enables the turnPID PIDController used for turning
+	 */
 	public void enableTurnPid() {
 		turnController.enable();
 	}
 
+	/**
+	 * Sets the setPoint of the turnPID PIDController
+	 * 
+	 * @param set
+	 *            The value to set the setPoint at
+	 */
```

Comment:
change method name por favor! like setTurnTarget. Same for setSetMove --> setDriveTarget (bc move is very general)